doc/problem/ultradian.md

```diff
@@ -18,15 +12,60 @@ Ultradian rhythm is widely observed in mammalian behavioural patterns. Ultradian
+    - Try to investigate cycles that are greater than 24 hours to avoid missing cycles.
+    - During the acclimatization period, investigate difference in cycles.
+- Plots
+    - Plots for determining optimal bin intervals for constructing the time series.
+    - Plots for discovering the frequency or period.
+    - General time series plots for getting intuitions for each variables.
+- Models
+    - Usage of Lomb-Scargle (LS) periodogram spectral analysis technique, a widely used tool in period detection and frequency analysis
+
+## Data Requirements Description
+- Input:  records for each strains (total of  16), each feature of interest (food, water, distance, active_state probability ...), in a duration of 12 days (excluding 4 acclimation days) . 
+- Processed: using one-minute time bins of movement records to binary score the activity into 0 (IS: inactive state) and 1 (AS: active state); using thirty-minute bins of food records to calculate the amount of chows consumed by mice; using LS periodogram technique to select the appropriate time bins for above.
+- Output: different patterned visualization for each feature, with the appropriate time bins that presents the most significant ultradian pattern.
+
+
+
 ## Method and models
 - Seasonal decomposition.
+    - Decomposition based on the rate of change.
+        - the Trend Component  that reflects the long term progression of the series (secular variation)
+        - the Cyclical Component  that describes repeated but non-periodic fluctuations
+        - the Seasonal Component  reflecting seasonality (seasonal variation)
+        - the Irregular Component  (or "noise") that describes random, irregular influences. It represents the residuals of the time series after the other components have been removed.
+
+    - Decomposition based on predictability.
 - Longitudinal data analysis.
-- Autocorrelation anslysis.
+    - Definition: A longitudinal study refers to an investigation where participant outcomes and possibly treatments or exposures are collected at multiple follow-up times.
+    -Strategies for Analyzing Longitudinal Data: 
+         - Traditional Repeated Measures ANOVA
+            - Mixed Models Anova
+            - Regression
+            - Multilevel Modeling
+
+- Autocorrelation analysis. 
+                - Autocorrelation, also known as serial correlation or cross-autocorrelation, is the cross-correlation of a signal with itself at different points in time (that is what the cross stands for). Informally, it is the similarity between observations as a function of the time lag between them. It is a mathematical tool for finding repeating patterns, such as the presence of a periodic signal obscured by noise, or identifying the missing fundamental frequency in a signal implied by its harmonic frequencies. It is often used in signal processing for analyzing functions or series of values, such as time domain signals.
```

Comment:
Remove this excerpt from wikipedia. Instead, you may want to explain how to perform the autocorrelation analysis using the data (e.g.: on what variables? what kind of autocorrelation analysis?).